### PR TITLE
[SV-COMP'18 14/19] Fixing member offset computation in presence of bitfields.

### PR DESCRIPTION
**Pull request by tautschnig**
~Do not merge: regression test needed.~

### CODE REVIEW

**Comment by tautschnig** [comment]
I have added a regression test, but the fix is either incomplete (more locations used to do the computation in the same style) or maybe not quite right.

**Review by marek-trtik — APPROVED** [review]
Looks good to me.

File: regression/cbmc/Bitfields3/main.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#include <stdlib.h>
+#include <assert.h>
+
+#pragma pack(1)
+struct A {
+  unsigned char a;
+  unsigned char b:2;
```
Comment (smowton) [inline regression/cbmc/Bitfields3/main.c]: You should probably also test the case where bitfields and "normal" fields are interleaved, e.g. `<byte, 2bit, byte, 2bit>`. Also the edge case where the bitfield precisely equals a byte (i.e. `<nybble, nybble>` should have size 1)

Comment (tautschnig) [reply regression/cbmc/Bitfields3/main.c]: Done.

**Comment by tautschnig** [comment]
@kroening @peterschrammel @smowton Ping?

**Review by smowton — APPROVED** [review]
Sorry for the delay

**Review by kroening — APPROVED** [review]

**Pull request merged**